yarn.lock

```diff
@@ -2777,13 +2772,6 @@
   dependencies:
     "@types/react" "*"
 
-"@types/react-google-recaptcha@^2.1.9":
```

Comment:
@types/react-google-recaptcha@^2.1.9 is also directly above this with the same integrity sha and resolved path, so it is not removed. Unsure how duplication could have gotten in